Extension/package.json

```diff
@@ -139,6 +139,12 @@
           "description": "Controls whether suspected compile errors detected by the IntelliSense engine will be reported back to the editor. Warnings about #includes that could not be located will always be reported to the editor. This setting is ignored by the Tag Parser engine.",
           "scope": "resource"
         },
+        "C_Cpp.dimInactiveRegions": {
+          "type": "boolean",
```

Comment:
Should we be forward looking and change this to an opacity setting, e.g. 100 would be equal to false? Then when VS Code enables real opacity this setting could be used to change the amount of opacity.

Reply:
I'm open to that, but has VSCode committed to giving us an opacity setting?

Reply:
I assume they would "eventually" implement text opacity to match VS 2017, it seems like adding an opacity setting now would confuse users since it wouldn't work (unless we make it affect the gray color).

Reply:
@bobbrow @sean-mcmanus Just bumping this while looking through this PR to learn how to do my own setting. As of 3 days ago an opacity property was exposed. https://github.com/Microsoft/vscode/issues/43004
It was merged into master, though I'm not sure when it will ship. As soon as it is, however, we can expose our own opacity setting as a 0-100 or similar as Sean suggested.